speech/grpc/transcribe_async_test.py

```diff
@@ -0,0 +1,26 @@
+# Copyright 2016, Google, Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import re
+
+from transcribe_async import main
+
+
+def test_main(resource, capsys, cloud_config):
+
+    # Run the transcribe sample on audio.raw, verify correct results
+    speech_storage_uri = 'gs://' + cloud_config.storage_bucket
```

Comment:
Lol okay, I promise I'm not doing this just to annoy you:

Prefer:

```python
storage_uri = 'gs://{}/speech/audio.raw'.format(cloud_config.storage_bucket)
```

(Here and in the other test file).